wgpu/src/lib.rs

```diff
@@ -2605,18 +2605,7 @@ impl<'a> RenderPass<'a> {
     ///
     /// The active vertex buffers can be set with [`RenderPass::set_vertex_buffer`].
     ///
-    /// The structure expected in `indirect_buffer` is the following:
-    ///
-    /// ```rust
-    /// #[repr(C)]
-    /// struct DrawIndirect {
-    ///     vertex_count: u32, // The number of vertices to draw.
-    ///     instance_count: u32, // The number of instances to draw.
-    ///     first_vertex: u32, // The Index of the first vertex to draw.
-    ///     first_instance: u32, // The instance ID of the first instance to draw.
-    ///     // has to be 0, unless [`Features::INDIRECT_FIRST_INSTANCE`] is enabled.
-    /// }
-    /// ```
+    /// The structure expected in `indirect_buffer` is a [`DrawIndirect`](crate::util::DrawIndirect).
```

Comment:
I'm a bit worried about this pedagogically that it might make it harder to figure out what is going on.

Otoh, people using indirect rendering generally should know what's going on.

Reply:
What do you think of:
> The structure expected in indirect_buffer must conform to DrawIndirect.
?

Reply:
yes, they should probably know :) I think it's ok, we can come back to this if needed. Resolving to make CI happy